src/d3d12/nvapi_d3d12_device.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <mutex>
```

Comment:
`<mutex>` is already included in `nvapi_private.h`, no need to include it again.